### ISSUE TITLE
500 zip

### ISSUE THREAD

**Issue by garyluu** [dockstore/dockstore#4405]
**Describe the bug**
Users (maybe bots) are encountering 500 errors when using the zip endpoint:

**To Reproduce**
Go to any one of these:
1. https://dockstore.org/api/workflows/16260/zip/81856
2. /api/workflows/16059/zip/57339
2. /api/workflows/16058/zip/57338
3. /api/workflows/16060/zip/57340

**Expected behavior**
Depending on what the real issue is. If it's not a valid version for the workflow, then it should just be a not found or bad request response.

**Screenshots**

**Additional context**
Kind of full log of one of them:
[01/Aug/2021:17:49:29 +0000] "GET /api/workflows/16060/zip/57340 HTTP/1.1" 500 110 "-" "Mozilla/5.0 (Linux; Android 6.0.1; Nexus 5X Build/MMB29P) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.90 Mobile Safari/537.36 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)"
[Webservice](https://github.com/dockstore/dockstore/releases/tag/1.11.5) - 1.11.5

[UI](https://github.com/dockstore/dockstore-ui2/releases/tag/2.8) - 2.8

[Compose Setup](https://github.com/dockstore/compose_setup/releases/tag/1.11) - 1.11

[Deploy](https://github.com/dockstore/dockstore-deploy/releases/tag/1.11.0) - 1.11.0

┆Issue is synchronized with this [Jira Story](https://ucsc-cgl.atlassian.net/browse/DOCK-1879)
┆Issue Number: DOCK-1879
┆Sprint: Sprint 74- Aquaman
┆Issue Type: Story

**Comment by denis-yuen** [comment]
Changing the milestone to reflect that this is in develop then

**Comment by coverbeck** [comment]
Belated thought: do we know if this request is coming from the UI, or is it a bots thing like @garyluu hypothesizes?

It's another bug if it's "genuinely" coming from the UI (e.g., not using Burp to intercept).

**Comment by svonworl** [comment]
Charles requested that I investigate further, so I looked at the prod web logs, and my conclusion is that most of the zip 500 errors are from bots or probes.  Probably the vast majority, but it's hard to definitively declare all of them as such.  Some have obvious bot User-Agents, others have "normal" User-Agents but happen in close temporal proximity to clear bot activity, none (that I've found) have Referer set.  Also, I confirmed that workflow 16260 currently exists in a recent db dump, but workflow version 81856 does not.
